driver-core/src/main/com/mongodb/internal/operation/ServerVersionHelper.java

```diff
@@ -44,6 +44,26 @@ public static boolean serverIsAtLeastVersionFourDotZero(final ConnectionDescript
         return description.getMaxWireVersion() >= 7;
     }
 
+    public static boolean serverIsLessThanVersionThreeDotZero(final ConnectionDescription description) {
```

Comment:
This should have been part of the next commit.  But I'm going to squash anyway, so not going to bother to change it now.